### DISCUSSION TITLE
Visit page path as goal conversion not reflecting

### DISCUSSION THREAD

**Discussion by JeremC7** [plausible/analytics#2448] [Anything Else]
I've set up a visit page path as a goal conversion but it does not show up in the stats. However, it does reflect as in the "top pages" visits. What am I doing wrong?

I've made a screencast in case it helps: https://tella.video/my-video-1-fu44

**Comment by metmarkosaric** [comment]
thanks for reporting that @JeremC7! could you try to add that first pageview goal once again please? add it as `/demo-confirmed-video` (basically an exact match to what we show in the pages report). please make sure to not have any blank space before or after the page path when adding the goal. sometimes people add a blank space which makes the goal not display

**Reply by JeremC7** [reply]
@metmarkosaric I've just deleted the initial goal, re-added it and it works just fine now. Thank you!
